CMakeLists.txt

```diff
@@ -18,7 +18,7 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 2.8)
```

Comment:
GNUInstallDirs requires cmake >= 2.8.5

Reply:
Ok, I will change to >=2.8.5 then. Any other hits?